tango_simlib/tango_launcher.py

```diff
@@ -21,41 +21,65 @@
 
-required_argument('--name', action='append',
-                  help="TANGO name(s) for the devices i.e.specified multiple times",)
-required_argument('--class', dest='device_class', action='append',
-                  help="TANGO class name(s) for the device(s) i.e. specified the " +
-                  "same number of times and the names and classes are matched in order")
-required_argument('--server-command', help="TANGO server executable command")
-required_argument('--server-instance', help="TANGO server instance name")
-required_argument('--port', help="TCP port where TANGO server should listen")
-parser.add_argument('--file-name', help="ASCII file containing device configuration parameters" +
-                    "and device network access parameter if TANGO DB server is not used")
-parser.add_argument('--put-device-property', action='append', help=
-                    "Put a device property into the TANGO DB, format is: "
-                    "'device/name/X:property_name:property_value'. Only allows "
-                    "properties to be set on devices started with this command. "
-                    "Can be specified multiple times.",
-                    dest='device_properties', default=[])
+required_argument(
+    "--name",
+    action="append",
+    help="TANGO name(s) for the devices i.e.specified multiple times",
+)
+required_argument(
+    "--class",
+    dest="device_class",
+    action="append",
+    help="TANGO class name(s) for the device(s) i.e. specified the "
+    + "same number of times and the names and classes are matched in order",
+)
+required_argument("--server-command", help="TANGO server executable command")
+required_argument("--server-instance", help="TANGO server instance name")
+required_argument("--port", help="TCP port where TANGO server should listen")
+parser.add_argument(
+    "--file-name",
+    help="ASCII file containing device configuration parameters"
+    + "and device network access parameter if TANGO DB server is not used",
+)
+parser.add_argument(
+    "--put-device-property",
+    action="append",
+    help="Put a device property into the TANGO DB, format is: "
+    "'device/name/X:property_name:property_value'. Only allows "
+    "properties to be set on devices started with this command. "
+    "Can be specified multiple times.",
+    dest="device_properties",
+    default=[],
+)
+
 
 def register_device(name, device_class, server_name, instance, db):
     dev_info = tango.DbDevInfo()
     dev_info.name = name
     dev_info._class = device_class
-    dev_info.server = "{}/{}".format(server_name.split('.')[0], instance)
-    print("""Attempting to register TANGO device {!r}
+    dev_info.server = "{}/{}".format(server_name.split(".")[0], instance)
+    print(
+        """Attempting to register TANGO device {!r}
     class: {!r}  server: {!r}.""".format(
-        dev_info.name, dev_info._class, dev_info.server))
+            dev_info.name, dev_info._class, dev_info.server
```

Comment:
The indenting here looks weird. Does the linter not complain?

Reply:
Black did the linting.
No complaints from flake8